packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -538,9 +538,7 @@ export class SecurityGroup extends SecurityGroupBase {
       // In the case of "allowAllOutbound", we don't add any more rules. There
       // is only one rule which allows all traffic and that subsumes any other
       // rule.
-      if (!remoteRule) { // Warn only if addEgressRule() was explicitely called
-        Annotations.of(this).addWarning('Ignoring Egress rule since \'allowAllOutbound\' is set to true; To add customized rules, set allowAllOutbound=false on the SecurityGroup');
-      }
+
```

Comment:
Code was removed, comments related to it were not.

Reply:
Removed the comments as well now.